docs/theme/partials/toc-item.html

```diff
@@ -1,16 +1,56 @@
+<!--
```

Comment:
This reverts the recent changes and re-introduces the issue that the TOC lacks detail about the option path. Let's keep experimentation with the TOC separate form this PR.

Reply:
Ah right - I had added that again before we discussed this as I found it useful for the pip docs (and still do ;)). But you are right that it shouldn't be in this PR. Dropped it.